app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class UserMailer < ApplicationMailer
+  def invitation_club(email, club)
+    @email = email
+    @club = club
+    mail(to: @email, subject: 'Invitation to a club') # rubocop:disable Rails/I18nLocaleTexts
```

Comment:
```suggestion
    @club = club
    mail(to: email, subject: 'Invitation to a club') # rubocop:disable Rails/I18nLocaleTexts
```

tu n'utilises pas cette variable d'instance